en/docs/develop/writing-a-custom-federated-authenticator.md

```diff
@@ -1,13 +1,18 @@
 # Writing a Custom Federated Authenticator
 
-It is possible to use the extension points available in the WSO2
-Identity Server to create custom federated authenticators.
+A custom federated authenticator can be written to authenticate a user with an external system. 
+So the external system can be any Identity provider such as Facebook, Twitter, Google and Yahoo. 
+It is possible to use the extension points available in the WSO2 Identity Server to create custom 
```

Comment:
```suggestion
You can use the extension points available in WSO2 Identity Server to create custom 
```

Reply:
Fixed https://github.com/wso2/docs-is/pull/2527/commits/ffc80dd6ac9088892baa7c56e5f03a9aa0768d3f